src/components/ColumnsEditor.tsx

```diff
@@ -6,7 +6,8 @@ import _ from 'lodash';
 import {CommandViewer} from './Commands';
 import {Command, CommandConfigT, defaultCommandConfig} from './CommandUtils';
 
-export function CommandDisplayer({filledCommands, style}) {
+
+export function CommandDisplayer({filledCommands, style}) {//
```

Comment:
CommandDisplayer is a poor name.  This component renders the JSON flavored LISP code that is sent to the interpreter.  It is mostly useful for me to debug stuff

Reply:
becomes `OperationDisplayer`